### PR TITLE
remove year calculation for financial changed dates

### PR DESCRIPTION
**Pull request by dreamfall**
…in DB

### CODE REVIEW

**Comment by dreamfall** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;77WtXoVu&#x2F;1069-qae16imp-financial-summary-date-error"> [QAE16IMP] Financial Summary - date error</a>

File: app/form_pointers/form_financial_pointer.rb [hunk 1]
```diff
@@ -203,8 +203,6 @@ def financial_year_changed_dates
 
       dates_by_years.each_with_index do |date, index|
         res << if (index + 1) != dates_by_years.count
-          date.join('/') + (last_year - (dates_by_years.count - (index + 1))).to_s
```
Comment (rusllonrails) [inline app/form_pointers/form_financial_pointer.rb]: @dreamfall 
I guess would be good to check other places there this block of code can be used before merge it.
1) Case Summary PDF
2) form PDF

**Comment by rusllonrails** [comment]
Looks good :+1: 
But would be good to check other places there this block of code can be used before merge it.
1) Case Summary PDF
2) form PDF

**Comment by rusllonrails** [comment]
Well done :+1:

**Pull request merged**